README.md

```diff
@@ -50,7 +51,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[residents:.pop[1]]
+castle[guests:.push=("Belle")]
```

Comment:
pop will always remove the last element in array. inorder to access the resdients array it would look something more like this:

```ruby
town[:residents]

# then in order to remove belle
town[:residents].delete("belle")

# then finally we could do something like this:
town[:castle].push(town[:residents].delete("belle"))
```